src/Sentry.Unity/WebGL/SentryWebGL.cs

```diff
@@ -0,0 +1,153 @@
+            // because Unity doesn't support async (multithreading) yet. This may change in the future so let's watch
+            // https://docs.unity3d.com/2019.4/Documentation/ScriptReference/PlayerSettings.WebGL-threadsSupport.html
+            options.BackgroundWorker = new WebBackgroundWorker(options, SentryMonoBehaviour.Instance);
+
+            // We may be able to do so after implementing the JS support.
+            // Additionally, we could recognize the situation when the unity gets stuck due to an error in JS/native:
+            //    "An abnormal situation has occurred: the PlayerLoop internal function has been called recursively.
+            //     Please contact Customer Support with a sample project so that we can reproduce the problem and troubleshoot it."
+            // Maybe we could write a file when this error occurs and recognize it on the next start. Like unity-native.
+            options.CrashedLastRun = () => false;
+        }
+    }
+
+    internal class WebBackgroundWorker : IBackgroundWorker
+    {
+        private readonly SentryMonoBehaviour _behaviour;
+        private readonly UnityWebRequestTransport _transport;
+
+        public WebBackgroundWorker(SentryUnityOptions options, SentryMonoBehaviour behaviour)
+        {
+            _behaviour = behaviour;
+            _transport = new UnityWebRequestTransport(options);
+        }
+
+        public bool EnqueueEnvelope(Envelope envelope)
+        {
+            _ = _behaviour.StartCoroutine(_transport.SendEnvelopeAsync(envelope));
+            return true;
+        }
+
+        public Task FlushAsync(TimeSpan timeout) => Task.CompletedTask;
+
+        public int QueuedItems { get; }
+    }
+
+    internal class UnityWebRequestTransport : HttpTransportBase
+    {
+        private readonly SentryUnityOptions _options;
+
+        public UnityWebRequestTransport(SentryUnityOptions options)
+            : base(options)
+        {
+            _options = options;
+        }
+
+        // adapted HttpTransport.SendEnvelopeAsync()
+        internal IEnumerator SendEnvelopeAsync(Envelope envelope)
+        {
+            using var processedEnvelope = ProcessEnvelope(envelope);
+            if (processedEnvelope.Items.Count > 0)
+            {
+                // Send envelope to ingress
+                var httpRequest = CreateRequest(processedEnvelope);
+                var www = CreateWebRequest(httpRequest);
+                yield return www.SendWebRequest();
+
+                var response = GetResponse(www);
+                if (response != null)
+                {
+                    HandleResponse(response, processedEnvelope);
+                }
+            }
+        }
+
+        private UnityWebRequest CreateWebRequest(HttpRequestMessage message)
+        {
+            // Note: In order to use the synchronous Envelope.Serialize() we ignore the `message.Content`
+            // which is an `EnvelopeHttpContent` instance and use the actual envelope it wraps.
```

Comment:
I think these comments are incorrect now and can be deleted.

Reply:
yes, accidental leftover, removing. Thanks